CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog
 
+### Next version
+
+- Fixed input data with a "count" column resulting in incorrect suppress bin
```

Comment:
```suggestion
- Fixed input data with a "count" column resulting in incorrect suppress bin.
```

Reply:
I updated my comment slightly. It's fine as it is. Though it sounds that you "fixed" the input data instead of the bug.